src/main/java/org/marketdesignresearch/mechlib/outcomerules/ccg/ConfigurableCCGFactory.java

```diff
@@ -54,15 +61,36 @@ public ConfigurableCCGFactory(NormFactory secondaryNormFactory, BlockingAllocati
 
 
     @Override
-    public CCGOutcomeRule getOutcomeRule(BundleValueBids<?> bids) {
+    public OutcomeRule getOutcomeRule(BundleValueBids<?> bids) {
+
+        BundleValueBids<?> originalBids = bids;
+
+        BigDecimal scalingFactor = null;
+        BigDecimal maxValue = bids.getBids().stream().map(BundleValueBid::getBundleBids).flatMap(Set::stream).map(BundleExactValuePair::getAmount).reduce(BigDecimal::max).orElse(BigDecimal.ZERO);
+        BigDecimal maxMipValue = BigDecimal.valueOf(MIP.MAX_VALUE).multiply(BigDecimal.valueOf(.8));
```

Comment:
I changed the factor to .8 as I had some instance in MRVM where an exception arised in the norm computation.

Reply:
:+1: